### DISCUSSION TITLE
Do the time series inputs need to be stationary for (mini)ROCKET?

### DISCUSSION THREAD

**Discussion by ebagjones** [timeseriesAI/tsai#248] [Q&A]
All,
This looks like a fantastic project and I really appreciate all the work put into the example notebooks.
I searched through everything and I can't find the answer, perhaps this means it is so obvious it should be understood by anyone using it. Apologies if this is the case...

Do my time series measurements in my multivariate datasets need to be stationary, or should they be stationary for best performance? I'm particularly interested in using MiniROCKET and while I see the data will be automatically normalized on input I wasn't clear about whether or not I needed to pre-process the data to make it stationary. 
Thanks in advance.

**Comment by oguiza** [comment]
>This looks like a fantastic project and I really appreciate all the work put into the example notebooks.

Thanks a lot @jefferyanderson!

No, there's no need to make data stationary. That's explained by the authors in their paper. 
Good luck with your experiments!

**Reply by ebagjones** [reply]
Wow, thanks for the fast response! This is an awesome project, thanks for all the time you put into it.
Have a great week.
